src/main/java/io/cdap/e2e/pages/locators/CdfPipelineRunLocators.java

```diff
@@ -74,8 +74,8 @@ public static WebElement locatePipelineStatus(String status) {
   @FindBy(how = How.XPATH, using = "(//*[contains(@class, 'metric-records-out-label')])[1]/following-sibling::span")
   public static WebElement recordsInCount;
 
-  @FindBy(how = How.XPATH, using = "(//*[contains(@class, 'metric-records-out-label')])[2]" +
-    "/following-sibling::span[text()!='0']")
+  @FindBy(how = How.XPATH, using = "//*[@data-type='batchsink']//*[contains(text(),'In')]" +
```

Comment:
The plugin might not be `batchsink` always. Can we use some better locator?

Reply:
Updated the locator and made it configurable. It will expect the plugin node name.